orthoinference/src/main/java/org/reactome/orthoinference/ReactionInferrer.java

```diff
@@ -122,55 +122,66 @@ public static void inferReaction(GKInstance reactionInst) throws InvalidAttribut
 							inferrableHumanEvents.add(reactionInst);
 							String inferredEvent = infReactionInst.getAttributeValue(DB_ID).toString() + "\t" + infReactionInst.getDisplayName() + "\n";	
 							Files.write(Paths.get(inferredFilehandle), inferredEvent.getBytes(), StandardOpenOption.APPEND);
-							logger.info("Successfully inferred " + reactionInst);
 						} else {
-							logger.info("\tCatalyst inference unsuccessful -- terminating inference for " + reactionInst);
+							logger.info("Catalyst inference unsuccessful -- terminating inference for " + reactionInst);
 						}
 					} else {
-						logger.info("\tOutput inference unsuccessful -- terminating inference for " + reactionInst);
+						logger.info("Output inference unsuccessful -- terminating inference for " + reactionInst);
 					}
 				} else {
-					logger.info("\tInput inference unsuccessful -- terminating inference for " + reactionInst);
+					logger.info("Input inference unsuccessful -- terminating inference for " + reactionInst);
 				}
 			} else {
-				logger.info("\tNo distinct proteins found in instance -- terminating inference for " + reactionInst);
+				logger.info("No distinct proteins found in instance -- terminating inference for " + reactionInst);
 			}
 		}
 	}
 	
 	// Function used to create inferred PhysicalEntities contained in the 'input' or 'output' attributes of the current reaction instance.
 	@SuppressWarnings("unchecked")
-	private static boolean inferReactionInputsOrOutputs(GKInstance reactionInst, GKInstance infReactionInst, String attribute) throws InvalidAttributeException, Exception
+	private static boolean inferReactionInputsOrOutputs(GKInstance reactionInst, GKInstance infReactionInst, String attribute) throws Exception
 	{
-		List<GKInstance> infAttributeInstances = new ArrayList<GKInstance>();
-		for (GKInstance attributeInst : (Collection<GKInstance>) reactionInst.getAttributeValuesList(attribute))
+		List<GKInstance> infAttributeInstances = new ArrayList<>();
+		Collection<GKInstance> attributeInstances = (Collection<GKInstance>) reactionInst.getAttributeValuesList(attribute);
+		logger.info("Total " + attribute + " instances: " + attributeInstances.size());
+		logger.info(attribute.substring(0,1).toUpperCase() + attribute.substring(1) + " instances: " + attributeInstances);
```

Comment:
So, `attribute.substring(0,1).toUpperCase() + attribute.substring(1)` is an abbreviation of attribute made of it's first two letters? Is it possible that this could get confusing if multiple attributes start with the same two letters, such as `goBiologicalProcess` and `goMolecularFunction`? Although I admit I'm not sure _which_ attributes will come through here.

Reply:
I just did that uppercase modification since the attributes `input` and `output` are lowercase, but I wanted the logging to look cleaner. They are the only attribute strings that are used here

Reply:
ok.